scripts/release_new_errata_records.py

```diff
@@ -0,0 +1,180 @@
+import argparse
+import asyncio
+import logging
+import os
+import sys
+import urllib.parse
+
+import aiohttp
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+    handlers=[
+        logging.StreamHandler(),
+    ],
+)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        "errata-releaser",
+        description="Releases updateinfo records through ALBS API",
+    )
+    parser.add_argument(
+        "-d",
+        "--date",
+        type=str,
+        default="2022-11-01",
+        required=False,
+        help="Date for start releasing (default: '2022-11-01')",
```

Comment:
Use [argparse.ArgumentDefaultsHelpFormatter formatter](http://docs.python.org/library/argparse.html#formatter-class) and argparse includes a default value automatically

https://stackoverflow.com/questions/12151306/argparse-way-to-include-default-values-in-help

Reply:
Thanks, fixed